src/main/java/billennium/tests/service/quiz/QuizServiceImpl.java

```diff
@@ -62,7 +63,7 @@ public QuizModel findQuizById(Long id) {
     public ResultModel checkAnswers(QuizModel quiz, List<Response> answersBundle) {
         ResultModel results = new ResultModel(answersBundle);
 
-        for (QuestionModel questionModel : quiz.getQuestionModels()) {
+        for (QuestionModel questionModel : quiz.getQuestionModels()) { // TODO nie da sie strumieniem ?
```

Comment:
Na pewno się da zapisać to bardziej funkcyjnie. Pisałeś może kiedyś w scali?

Reply:
tak i nie chce tego więcej robić.